### PR TITLE
create new interoperability section

### PR DESCRIPTION
**Pull request by eshaben**
### Description

- Create `Interoperability` section 
- Move `XCM` content to `Interoperability` section
- Move `integrations/bridges` content to `Interoperability` section and rename to `Cross-Chain Protocols`
- Update image paths and index page icons
- Update internal links so redirects aren't required, this prevents 302 status codes (for SEO purposes)

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [x] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [x] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

mkdocs pr: https://github.com/PureStake/moonbeam-mkdocs/pull/96
cn pr: https://github.com/PureStake/moonbeam-docs-cn/pull/201

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

n/a

### CODE REVIEW

File: builders/interoperability/xcm/remote-evm-calls.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ To this end, the [Ethereum-XCM pallet](https://github.com/PureStake/moonbeam/tre
 
 The happy path for both regular and remote EVM calls through XCM is portrayed in the following diagram:
 
-![Happy parth for regular and remote EVM calls through XCM](/images/builders/xcm/remote-evm-calls/xcmevm-1.png)
+![Happy parth for regular and remote EVM calls through XCM](/images/builders/interoperability/xcm/remote-evm-calls/xcmevm-1.png)
```
Comment (themacexpert) [inline builders/interoperability/xcm/remote-evm-calls.md:22]:
```suggestion
![Happy path for regular and remote EVM calls through XCM](/images/builders/interoperability/xcm/remote-evm-calls/xcmevm-1.png)
```

File: builders/interoperability/xcm/xc20/xtokens.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ description: Learn how to send XC-20s to other chains using the X-Tokens pallet.
 
 # Using the X-Tokens Pallet To Send XC-20s
 
-![X-Tokens Precompile Contracts Banner](/images/builders/xcm/xc20/xtokens/xtokens-banner.png)
+![X-Tokens Precompile Contracts Banner](/images/builders/interoperability/xcm/xc20/xtokens/xtokens-banner.png)
 
 ## Introduction {: #introduction }
```
Comment (themacexpert) [inline builders/interoperability/xcm/xc20/xtokens.md:11]: Outside of scope of review but while we're at it, line 35 has typo here "curriencies"

**Review by themacexpert — APPROVED** [review]
Looks good, 2 minor comments

File: builders/interoperability/xcm/xc20/xtokens.md (hunk 2 above)
Comment (eshaben) [reply builders/interoperability/xcm/xc20/xtokens.md:11]: good catch!

**Pull request merged**